src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -1114,10 +1114,9 @@ Outcome TwitchMessageBuilder::tryAppendEmote(const EmoteName &name)
 }
 
 std::optional<EmotePtr> TwitchMessageBuilder::getTwitchBadge(
-    const Badge &badge) const
+    const Badge &badge, const TwitchChannel *chan)
```

Comment:
nit: Rename the `TwitchChannel*` parameter to `twitchChannel` and make this a static function in the anonymous namespace (within `namespace { ... }`) at the top of the file